tests/src/test/scala/cats/tests/AlgebraInvariantSuite.scala

```diff
@@ -237,11 +268,6 @@ class AlgebraInvariantSuite extends CatsSuite {
   checkAll("Invariant[Monoid]", InvariantTests[Monoid].invariant[Option[MiniInt], Boolean, Boolean])
   checkAll("Invariant[Monoid]", SerializableTests.serializable(Invariant[Monoid]))
 
-  Eq[Band[Set[Boolean]]]
-  cats.laws.discipline.ExhaustiveCheck[Set[Boolean]]
-  Eq[(Set[Boolean], Boolean)]
-  Eq[(Set[Boolean], Set[Boolean] => (Set[Boolean], Boolean))]
-  Eq[CommutativeSemigroup[Set[Boolean]]]
```

Comment:
There was a [question](https://github.com/typelevel/cats/pull/3222#discussion_r360649967) about these lines in the previous PR. I've removed them but happy to revert if they are necessary.

Reply:
They might be compilation tests, no?

Reply:
On second thought, they probably don't belong there at all. So let's keep them removed.